### DISCUSSION TITLE
Unexpected scrolling of elements when dragging items of Sortable component

### DISCUSSION THREAD

**Discussion by Lumelomser** [uikit/uikit#4495] [Q&A] [answered]
The Sortable component is calling the `scrollParents()` function from utils while dragging items.

https://github.com/uikit/uikit/blob/4cb34fa89d66472a6942b49925283efea32d2ae9/src/js/components/sortable.js#L303

In UIkit v3.5.10 the default value of the `overflowRe` parameter was defined as `/auto|scroll/`.

https://github.com/uikit/uikit/blob/80bee05cceba07a5ffdece96457192d673607b7a/src/js/util/viewport.js#L123

In UIkit v3.5.11 the default value has changed to `/auto|scroll|hidden/`.

https://github.com/uikit/uikit/blob/6fd2025cc92244a411276863938a27ae23280054/src/js/util/viewport.js#L123

Because of this, I now have issues using the Sortable component.

If there are elements on the page that are explicitly defined with `overflow: hidden`, they will also start scrolling when a draggable element is dragged over them, although this is not desired.

Is this an intended behavior or was it just forgotten to pass the `/auto|scroll/` parameter when calling `scrollParents()` while refactoring?

```
scrollParents(document.elementFromPoint(x, pos.y), /auto|scroll/).reverse().some(scrollEl => { ... });
```

Before the Sortable component used the `scrollParents()` function from utils, `/auto|scroll/` was also used as default parameter.

https://github.com/uikit/uikit/blob/c4d909c4ea98b6207a29a0c96e3b87b1b0c025bc/src/js/components/sortable.js#L325-L330

I have created a small example that illustrates the described behavior.
https://codepen.io/lumelomser/pen/zYwbjeW

Steps to Reproduce the Problem
1. Open reproduction link.
2. Start dragging a sortable item up or down the page.
3. As soon as a sortable item is over one of these blue boxes, they start scrolling, even if `overflow: hidden` is defined.

@janschoenherr
Perhaps you may be able to provide more details?

**Comment by janschoenherr** [comment]
Thanks, I think you are probably right. Would you do a Pull Request?

**Comment by janschoenherr** [comment, marked as answer]
Fixed with https://github.com/uikit/uikit/commit/59737a9b5504a16023d12456d7456e5dde78771c

**Reply by Lumelomser** [reply]
Thank you, @janschoenherr !
